doc/source/contributing.rst

```diff
@@ -1077,5 +1077,78 @@ The branch will still exist on GitHub, so to delete it there do::
 
     git push origin --delete shiny-new-feature
 
+Sharing Docstrings
```

Comment:
can you add a ref here

Reply:
Will do. Going to wait till the docstring guide is merged and I move it to that document so that the name makes sense.